### DISCUSSION TITLE
OpenAI compatible minimal API using ORT-GenAI

### DISCUSSION THREAD

**Discussion by bpietroiu** [microsoft/onnxruntime-genai#995] [Show and tell]
I have created a minimal implementation, including a full port of a Jinja parser/interpreter 

Docker and docker compose for deployment
OpenTelemetry for logging and monitoring
Dynamic Batching 
Chat prompt template processing using Jinja JS to C# ported by me

can be found here https://github.com/bpietroiu/GenAI.Server

**Comment by kunal-vaishnavi** [comment]
Great work! You can also try out [Foundry Local](https://foundrylocal.ai/), which is powered by ONNX Runtime GenAI.
